depmapomics/config_dev.py

```diff
@@ -304,6 +304,22 @@
 WESCNWORKSPACE = "broad-firecloud-ccle/DepMap_WES_CN_hg38"
 WESMUTWORKSPACE = "broad-firecloud-ccle/DepMap_Mutation_Calling_CGA_pipeline"
 
+
+############## Fingerprinting
+
+# Local directory to save intermediate files to
+WORKING_DIR = "temp/"
+
+FPWORKSPACE = "broad-firecloud-ccle/CCLE_SNP_QC"
```

Comment:
wouldn't we want to use some temp workpaces for the dev branch? (same for WESCNWORKSPACE/ WESMUTWORKSPACE )